tests/kernel/pipeline_tests.py

```diff
@@ -40,9 +40,8 @@ class TestCases(unittest.TestCase):
   sample_cell_body = """
 email: foo@bar.com
 schedule:
-  start_date: Jun 1 2005  1:33PM
-  end_date: Jun 10 2005  1:33PM
-  datetime_format: '%b %d %Y %I:%M%p'
+  start_date: 2009-05-05T22:28:15Z
```

Comment:
These test lines, and some below, include a Z, but I don't see Z used in the format in the main code. Do these tests pass? Do they also pass without the Z, and do they fail if you add some other junk after the Z?

Reply:
The tests pass because yaml.load() will successfully parse 2009-05-05T22:28:15Z. For now, they will pass without the Z as well, but not after https://b.corp.google.com/issues/64951979#comment2 is fixed.